lib/c-api/README.md

```diff
@@ -1,136 +1,2 @@
 # `wasmer-c-api` [![Build Status](https://github.com/wasmerio/wasmer/workflows/build/badge.svg?style=flat-square)](https://github.com/wasmerio/wasmer/actions?query=workflow%3Abuild) [![Join Wasmer Slack](https://img.shields.io/static/v1?label=Slack&message=join%20chat&color=brighgreen&style=flat-square)](https://slack.wasmer.io) [![MIT License](https://img.shields.io/github/license/wasmerio/wasmer.svg?style=flat-square)](https://github.com/wasmerio/wasmer/blob/master/LICENSE)
 
-This crate exposes a C and a C++ API for the Wasmer runtime. It also fully supports the [wasm-c-api common API](https://github.com/WebAssembly/wasm-c-api).
-
-##  Usage
-
-Once you [install Wasmer in your system](https://github.com/wasmerio/wasmer-install), the *shared object files* and the *headers* will be automatically available **inside the Wasmer installed path**.
-
-The C ([`wasmer.h`][wasmer_h]) and C++ ([`wasmer.hh`][wasmer_hh]) header
-files can be found in the Wasmer `include` directory:
-
-```bash
-wasmer config --includedir
-```
-
-The runtime shared libraries (`.so`, `.dylib`, `.dll`) can be found in the Wasmer
-`lib` directory:
-
-```bash
-wasmer config --libdir
-```
-
-> Note: You can also download the libraries or header files directly
-from [Wasmer release page].
-
-The full C API documentation can be found here:
-https://wasmerio.github.io/wasmer/c-api/
```

Comment:
We should keep this. It's going to be useful regardless

Reply:
I've removed it because it might outdated. It's a reminder for me to not forget to update that part. The PR is still a work in progress :-).